etc/eslint/overrides/index.js

```diff
@@ -155,6 +160,10 @@ var overrides = [
 			'vars-on-top': 'off',
 			'node/no-unpublished-require': 'off'
 		}
+	},
+	{
+		'files': ['*.d.ts', '*.ts', '*.tsx'],
```

Comment:
```suggestion
		'files': [ '*.d.ts', '*.ts', '*.tsx' ],
```

Reply:
@Planeshifter Is `*.tsx` relevant for us? We don't currently support `*.jsx`, so not clear why we'd include `*.tsx` here.